### PR TITLE
Add ability to configure "huge" PR threshold

### PR DESCRIPTION
**Pull request by rajanand02**
Adds a new config to set the limit for a huge PR comment

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -28,17 +28,24 @@ const getInputs = () => {
   const GITHUB_TOKEN: string = core.getInput('github-token', { required: true });
   const BRANCH_IGNORE_PATTERN: string = core.getInput('skip-branches', { required: false }) || '';
   const SKIP_COMMENTS: string = core.getInput('skip-comments', { required: false }) || 'false';
+  const PR_THRESHOLD: string = core.getInput('pr-threshold', { required: false }) || '';
```
Comment (rajanand02) [inline src/main.ts]: @rheaditi Feel free to suggest a better and more relatable name

**Comment by Siddharth11** [comment]
LGTM 👍

File: __tests__/utils.test.ts [hunk 2]
```diff
@@ -28,7 +32,7 @@ describe('shouldSkipBranchLint()', () => {
     expect(shouldSkipBranchLint('foo', '[0-9]{2}')).toBeFalsy();
     expect(shouldSkipBranchLint('f00', '[0-9]{2}')).toBeTruthy();
 
-    const customBranchRegex = '^(production-release|master|release\/v\\d+)$';
+    const customBranchRegex = '^(production-release|master|release/v\\d+)$';
```
Comment (rheaditi) [inline __tests__/utils.test.ts]:
```suggestion
    const customBranchRegex = '^(production-release|master|release\/v\\d+)$';
```

File: __tests__/utils.test.ts [hunk 3]
```diff
@@ -201,3 +205,43 @@ describe('getStoryTypeLabel()', () => {
     expect(getStoryTypeLabel({} as any)).toEqual('');
   });
 });
+
+describe('isHumongousPR()', () => {
+  it('should return true if additions are greater than the threshold', () => {
+    expect(isHumongousPR(2000, 500)).toBeTruthy();
+  });
+
+  it('should return false if additions are lesser than the threshold', () => {
```
Comment (rheaditi) [inline __tests__/utils.test.ts]: :P 

```suggestion
  it('should return false if additions are less than the threshold', () => {
```

File: src/main.ts (hunk 1 above)
Comment (rheaditi) [reply src/main.ts]: This is good :+1:

File: src/main.ts [hunk 4]
```diff
@@ -28,17 +28,24 @@ const getInputs = () => {
...
+    PR_THRESHOLD,
   };
 };
 
 async function run() {
   try {
-    const { PIVOTAL_TOKEN, GITHUB_TOKEN, BRANCH_IGNORE_PATTERN, SKIP_COMMENTS } = getInputs();
+    const { PIVOTAL_TOKEN, GITHUB_TOKEN, BRANCH_IGNORE_PATTERN, SKIP_COMMENTS, PR_THRESHOLD } = getInputs();
+
+    const defaultAddtionsCount = 800;
```
Comment (rheaditi) [inline src/main.ts]: `defaultAdditionsCount` :P

File: src/utils.ts [hunk 5]
```diff
@@ -340,15 +340,15 @@ ${body}`;
  * @param {number} addtions
  * @return {boolean}
  */
-export const isHumongousPR = (additons: number): boolean => additons > 1000;
+export const isHumongousPR = (additons: number, threshold: number): boolean => additons > threshold;
```
Comment (rheaditi) [inline src/utils.ts]: `additions` :P

**Pull request merged**